### PR TITLE
Initial Resmo Integration

### PR DESCRIPTION
**Pull request by shuetisha**
### What does this PR do?

Adds an Resmo integration to Datadog

### Motivation

The Resmo is a continuous asset visibility, security, and compliance solution for cloud and SaaS. Integrate with your tools to start querying and securing your resources. Monitor them in one place, run security and compliance audits, and get alerted on violations.
Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.

What does Resmo offer to Datadog users?
- Collect and monitor all your Datadog assets on a single platform.
- Query your Datadog monitors, roles, permissions, API Keys, and more.
- Set up rules to continuously assess your Datadog resources.
- Set up and receive rule notifications based on your Datadog resource configurations.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by maycmlee** [comment]
Created [DOCS-5013](https://datadoghq.atlassian.net/browse/DOCS-5013) for docs review.

File: resmo/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+[2]: https://github.com/DataDog/integrations-extras/blob/master/resmo/assets/images/integrations.png
+[3]: https://github.com/DataDog/integrations-extras/blob/master/resmo/assets/images/setup-integration.png
+[4]: https://github.com/DataDog/integrations-extras/blob/master/resmo/assets/images/datadog-application-keys.png
+[5]: https://github.com/DataDog/integrations-extras/blob/master/resmo/assets/images/datadog-new-key.png
+[6]: https://github.com/DataDog/integrations-extras/blob/master/resmo/assets/images/datadog-created-key.png
+[7]: https://github.com/DataDog/integrations-extras/blob/master/resmo/assets/images/resmo-key-setup.png
+[8]: https://github.com/DataDog/integrations-extras/blob/master/resmo/assets/images/resmo-site-setup.png
+[9]: https://docs.resmo.com/resources/datadog
+[10]: https://www.resmo.com/contact
+[11]: https://docs.datadoghq.com/help/
```
Comment (buraizu) [inline resmo/README.md:66]:
```suggestion
[11]: https://docs.datadoghq.com/help/
[12]: https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token
[13]: https://docs.datadoghq.com/account_management/api-app-keys/#add-application-keys
```

File: resmo/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- **In Resmo**: Go to https://resmo.com, login and navigate to **Integrations**.<br/>
+
+![Resmo Screenshot][2]
+
+- Click **Add Integration** and select **Datadog**.<br />
+- Write a descriptive name and optionally a description.
+
+![Resmo Screenshot][3]
+
+- Then, go to your Datadog account and create a new organization API key and a user Application key.<br />
```
[the remark this answers is not in the document]
Comment (buraizu) [reply resmo/README.md:23]: As the instructions for creating these resources already exist at the new links added here, there's no need to duplicate them in this document.

File: resmo/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  - Step 1: Go to Organization Settings>Application Keys on Datadog. <br />
+![Resmo Screenshot][4]
+  - Step 2: Click the +New Key button to create a key. Name your key and hit Create. <br />
+![Resmo Screenshot][5]
+  - Step 3: Find your key in the Application Keys tab. Then, copy it. <br />
+![Resmo Screenshot][6]
+
+- Return to your Resmo account. Enter the copied keys into the related key fields (Organization API Key and User App Key). <br />
```
Comment (buraizu) [inline resmo/README.md:31]:
```suggestion
4. Return to your Resmo account and enter the copied keys into the Organization API Key and User App Key fields. <br />
```

File: resmo/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+![Resmo Screenshot][7]
+
+- Select a site from the dropdown menu on the last field.
+
+![Resmo Screenshot][8]
+
+-Hit the create button, and that's it! You're ready to query your Datadog resources.
```
Comment (buraizu) [inline resmo/README.md:39]:
```suggestion
6. Click the **create** button, and you're ready to query your Datadog resources.
```

File: resmo/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+## Uninstallation
+
+- **In Resmo**: Go to Settings>Integrations and select the Datadog integration you wish to uninstall. <br />
```
Comment (buraizu) [inline resmo/README.md:43]:
```suggestion
- **In Resmo**: Go to Settings>Integrations and select the Datadog integration you wish to uninstall. <br />
```
```suggestion
- **In Resmo**: Go to **Settings > Integrations** and select the Datadog integration you wish to uninstall. <br />
```

File: resmo/README.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+- You have two options;
+  - To temporarily pause the integration, click the Disable button.
+  - To permanently remove the integration, click the Delete button.
```
Comment (buraizu) [inline resmo/README.md:47]:
```suggestion
  - To temporarily pause the integration, click the **Disable** button.
  - To permanently remove the integration, click the **Delete** button.
```

File: resmo/README.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+## Data Collected
+
+See [Resmo Datadog Resources][9] for the list of resources we collect from your Datadog account.
```
Comment (buraizu) [inline resmo/README.md:51]:
```suggestion
See [Resmo Datadog Resources][9] for the list of resources collected from your Datadog account.
```

File: resmo/README.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+## Support
+Need help? Contact [Resmo support][10] or [Datadog support][11]
```
Comment (buraizu) [inline resmo/README.md:54]:
```suggestion
Need help? Contact [Resmo support][10] or [Datadog support][11].
```

File: resmo/README.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+
+## Setup
+
+- **In Resmo**: Go to https://resmo.com, login and navigate to **Integrations**.<br/>
+
+![Resmo Screenshot][2]
+
+- Click **Add Integration** and select **Datadog**.<br />
+- Write a descriptive name and optionally a description.
```
Comment (buraizu) [inline resmo/README.md:19]:
```suggestion
2. Click **Add Integration** and select **Datadog**. Write a descriptive name and optionally a description.<br />
```

File: resmo/README.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+  - Step 2: Click the +New Key button to create a key. Name your key and hit Create. <br />
+![Resmo Screenshot][5]
+  - Step 3: Find your key in the Application Keys tab. Then, copy it. <br />
+![Resmo Screenshot][6]
+
+- Return to your Resmo account. Enter the copied keys into the related key fields (Organization API Key and User App Key). <br />
+
+![Resmo Screenshot][7]
+
+- Select a site from the dropdown menu on the last field.
```
Comment (buraizu) [inline resmo/README.md:35]:
```suggestion
5. Select a site from the dropdown menu on the last field.
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the pull request, and nice work on the documentation!
I left some minor suggestions based on our style guide:
https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md
Please let me know if you have any questions.

File: resmo/README.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+4. Return to your Resmo account and enter the copied keys into the Organization API Key and User App Key fields. <br />
+
+![Resmo Screenshot][7]
+
+5. Select a site from the dropdown menu on the last field.
+
+![Resmo Screenshot][8]
+
+6. Click the **create** button, and you're ready to query your Datadog resources.
+
```
Comment (buraizu) [inline resmo/README.md:33]:
```suggestion
## Data Collected

See [Resmo Datadog Resources][9] for the list of resources collected from your Datadog account.

```

Comment (buraizu) [reply resmo/README.md:33]: Optional suggestion to move the **Data Collected** section above **Uninstallation** -- I think that users will want to see the value of the integration after it's installed, and then learn how to uninstall if needed -- but that's just my opinion, so feel free to ignore this suggestion if this format is intentional for some other reason.

File: resmo/README.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+- **In Resmo**: Go to **Settings > Integrations** and select the Datadog integration you wish to uninstall. <br />
+
+- You have two options;
+  - To temporarily pause the integration, click the **Disable** button.
+  - To permanently remove the integration, click the **Delete** button.
+
+## Data Collected
+
+See [Resmo Datadog Resources][9] for the list of resources collected from your Datadog account.
```
Comment (buraizu) [inline resmo/README.md:44]:
```suggestion
```

Comment (buraizu) [reply resmo/README.md:44]: As mentioned above, this is an optional suggestion to move this section.

File: resmo/README.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+
+6. Click the **create** button, and you're ready to query your Datadog resources.
+
+## Uninstallation
+
+- **In Resmo**: Go to **Settings > Integrations** and select the Datadog integration you wish to uninstall. <br />
+
+- You have two options;
+  - To temporarily pause the integration, click the **Disable** button.
+  - To permanently remove the integration, click the **Delete** button.
```
Comment (buraizu) [inline resmo/README.md:40]:
```suggestion
  1. To temporarily pause the integration, click the **Disable** button.
  2. To permanently remove the integration, click the **Delete** button.
```

File: resmo/README.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+
+![Resmo Screenshot][8]
+
+6. Click the **create** button, and you're ready to query your Datadog resources.
+
+## Uninstallation
+
+- **In Resmo**: Go to **Settings > Integrations** and select the Datadog integration you wish to uninstall. <br />
+
+- You have two options;
```
Comment (buraizu) [inline resmo/README.md:38]:
```suggestion
- You have two options:
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thank you for applying that feedback. I've left some additional feedback and some optional suggestions. 
Also, it looks like a check is failing due to images not being checked into the repo under the `resmo/images` folder. Would you be able to include the images?
Thanks again, and let me know if you have any questions.

**Pull request closed without merging**